src/App/Exercises/HtmlCss/ArrayRendering/index.jsx

```diff
@@ -0,0 +1,37 @@
+import './styles.css';
+import { Data } from './Data';
+
+export const ArrayRendering = () => {
+  return (
+    <div>
+      <table className="band__table">
+        <thead>
+          <tr>
+            <th>Band</th>
+            <th>Year Formed</th>
+            <th>Albums</th>
+            <th>Most Famous Song</th>
+          </tr>
+        </thead>
+        {Data.map(({ band, yearFormed, albums, mostFamousSong }) => {
+          return (
+            <tbody key={band}>
```

Comment:
Fajnie, że dodałeś również <thead>, <tbody> i <tfoot>. Nie wspominałem o nich, bo nie są obowiązkowe, a dzisiejsze przeglądarki już to sobie dodają same. Jednak w Twoim przypadku będziesz generować kilka tych tagów `<tbody>`, co już nie będzie zgodne z implementacją HTML. Powinnaś tutaj generować "tr-ki", a nie "tbody".

Reply:
Normalnie bym tego nie dodało ale zauważyłam że w chromie consola mi wykrzyczała że tego chce to dopisałam bo nie lubie mieć warningów ani errorów nigdzie 😅